app/uk/gov/hmrc/cataloguefrontend/platforminitiatives/PlatformInitiativesListPage.scala.html

```diff
@@ -59,9 +59,10 @@ <h1>Platform Initiatives@if(team!=""){: @team}</h1>
             <input class="search form-control" id="search" type="text">
         </div>
         <div class="right padding-reset-right" style="line-height:2.2em;">
-                <a href="@appRoutes.PlatformInitiativesController.platformInitiatives(DisplayType.Chart)">Pie chart</a> |
-                <a href="@appRoutes.PlatformInitiativesController.platformInitiatives(DisplayType.Progress)">Progress bar</a>
-
+                <a href="@appRoutes.PlatformInitiativesController.platformInitiatives(DisplayType.Chart)">
+                    @if(display == DisplayType.Chart){<strong>Pie chart</strong>}else{Pie chart}</a> |
```

Comment:
if "Pie chart" and "Progress bar" was moved to a field of `DisplayType` the code would be more maintainable - something like
```scala
Seq(DisplayType.Chart, DisplayType.Progress).map { dt =>
  <a href="@platformInitiatives(dt)> @(if(display == dt){ <strong>@dt</strong> } else @dt }
}
```